.github/workflows/publish_charm.yaml

```diff
@@ -0,0 +1,21 @@
+name: Publish to edge
+
+# On push to a "special" branch, we:
+# * always publish to charmhub at latest/edge/branchname
+# * always run tests
+# where a "special" branch is one of main/master or track/**, as
```

Comment:
Only `main` is targeted here

Reply:
Updated the comments, thank you! I'll issue a PR against the charms requiring the comment changes as well :)